File: dotnet/tests/amp.tests/messaging/NullHandler.cs
using System;
using System.Collections.Generic;
using amp.messaging;

namespace amp.tests.messaging
{
    class NullHandler : IMessageHandler
    {
        public string Topic
        {
            get { return "NULL"; }
        }

        public object Handle(object message, IDictionary<string, string> headers)
        {
            return null;
        }

        public object HandleFailed(cmf.bus.Envelope env, Exception ex)
        {
            return null;
        }
    }
}

File: java/amp-rabbit/src/test/java/amp/messaging/NullHandler.java
package amp.messaging;

import java.util.Map;

import cmf.bus.Envelope;

public class NullHandler implements IMessageHandler<Object> {

	@Override
	public Class<Object> getMessageType() {
		return Object.class;
	}

	@Override
	public Object handle(Object message, Map<String, String> headers) {
		return null;
	}

	@Override
	public Object handleFailed(Envelope envelope, Exception e) {
		return null;
	}

}
